### PR TITLE
[vrops-exporter] added prefix info to config

### PR DESCRIPTION
**Pull request by richardtief**

### CODE REVIEW

File: prometheus-exporters/vrops-exporter/templates/collector-configmap.yaml [hunk 1]
```diff
@@ -22,6 +22,7 @@ data:
     #    - 'VMStatsDefaultCollector'
 
     ClusterPropertiesCollector:
+    # Prefix: vrops_cluster_
```
Comment (viennaa) [inline prometheus-exporters/vrops-exporter/templates/collector-configmap.yaml:25]: why is it commented out?

**Review by christopherhans — APPROVED** [review]
Good for those who are new to the exporter.

**Comment by viennaa** [comment]
If it is an information, please specify it as such. looks like it is a configurable thing in the config. or is it?

File: prometheus-exporters/vrops-exporter/templates/collector-configmap.yaml (hunk 1 above)
Comment (richardtief) [reply prometheus-exporters/vrops-exporter/templates/collector-configmap.yaml]: Just an info. Added `INFO` to the comment.

**Pull request merged**